app/core/src/main/java/com/fsck/k9/AccountPreferenceSerializer.kt

```diff
@@ -174,13 +161,14 @@ class AccountPreferenceSerializer(
             val description = storage.getString("$accountUuid.$IDENTITY_DESCRIPTION_KEY.$ident", null)
             val replyTo = storage.getString("$accountUuid.replyTo.$ident", null)
             if (email != null) {
-                val identity = Identity()
-                identity.name = name
-                identity.email = email
-                identity.signatureUse = signatureUse
-                identity.signature = signature
-                identity.description = description
-                identity.replyTo = replyTo
+                val identity = Identity(
+                    name = name,
```

Comment:
It feels redundant to use named arguments when the variable names are named exactly like the parameters.

Reply:
Hm, yeah maybe. I did it like this coming from `copy`, since it allows passing the arguments without thinking about the order or default arguments